apis/test_config.py

```diff
@@ -90,6 +89,31 @@ def name(self):
     return f'v{self.version}-{self.cores}'
 
 
+# We may want to use GKE in the future.
+@attrs.define
+class Gpu(Accelerator):
+  """Represents a single Cloud GPU instance.
+
+  Attributes:
+    machine_type: The host type of the GPU. E.g., `a2-highgpu-1g`.
+    image_family: Family of the image.
+    count: number of the GPU devices.
+    accelerator_type: Type of the accelerator. E.g., `nvidia-test-v100`.
+    runtime_version: Runtime image version.
+  """
+
+  machine_type: str
+  image_family: str
+  count: str
+  accelerator_type: str
+  runtime_version: str
+
+  @property
+  def name(self):
```

Comment:
Could you help me understand more here? could we directly call `self.accelerator_type` if there is no extra processing step?

Reply:
I leave the name there in case we want something like `nvidia-tesla-v100-4` or just `v100` instead of `nvidia-tesla-v100` in the future. WDYT?

Reply:
I see. Probably we could add a TODO comment, or we could directly call `self.accelerator_type` and update it in the future when we are clear on names?